cli/src/cli.rs

```diff
@@ -337,7 +337,15 @@ pub async fn new_wallet(cli: Cli) -> Result<Option<Wallet>, Error> {
             Some(wallet)
         } else {
             // init new wallet with default init parameters
-            let init_params = InitParameters::default();
+            let mut init_params = InitParameters::default();
+
+            if let Some(stronghold_snapshot_path) = std::env::var("STRONGHOLD_SNAPSHOT_PATH").ok() {
+                init_params.stronghold_snapshot_path = stronghold_snapshot_path;
+            }
+            if let Some(node_url) = std::env::var("NODE_URL").ok() {
```

Comment:
Use if let Ok() without `.ok()`?

Reply:
I don't mind changing if you insist, but I did it like this for readability as it reads like: "If there's (some) stronghold snapshot path" in the `.env` file, then use it. `if let Ok(...)` reads worse imo.

Reply:
To me it just looks more confusing with the useless `.ok()`

Reply:
I'd agree to change it

Reply:
Changed

Reply:
Thanks